xchange-bitfinex/src/main/java/org/knowm/xchange/bitfinex/v2/dto/marketdata/BitfinexTicker.java

```diff
@@ -2,104 +2,50 @@
 
 import com.fasterxml.jackson.annotation.JsonFormat;
 import java.math.BigDecimal;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
 
 @JsonFormat(shape = JsonFormat.Shape.ARRAY)
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
 public class BitfinexTicker {
 
   private String symbol;
+  private BigDecimal frr;
   private BigDecimal bid;
+  private BigDecimal bidPeriod;
   private BigDecimal bidSize;
   private BigDecimal ask;
+  private BigDecimal askPeriod;
   private BigDecimal askSize;
   private BigDecimal dailyChange;
   private BigDecimal dailyChangePerc;
   private BigDecimal lastPrice;
   private BigDecimal volume;
   private BigDecimal high;
   private BigDecimal low;
-
-  public BitfinexTicker() {}
-
-  public BitfinexTicker(
-      String symbol,
-      BigDecimal bid,
-      BigDecimal bidSize,
-      BigDecimal ask,
-      BigDecimal askSize,
-      BigDecimal dailyChange,
-      BigDecimal dailyChangePerc,
-      BigDecimal lastPrice,
-      BigDecimal volume,
-      BigDecimal high,
-      BigDecimal low) {
-
-    this.symbol = symbol;
-    this.bid = bid;
-    this.bidSize = bidSize;
-    this.ask = ask;
-    this.askSize = askSize;
-    this.dailyChange = dailyChange;
-    this.dailyChangePerc = dailyChangePerc;
-    this.lastPrice = lastPrice;
-    this.volume = volume;
-    this.high = high;
-    this.low = low;
-  }
-
-  public String getSymbol() {
-    return symbol;
-  }
-
-  public BigDecimal getBid() {
-    return bid;
-  }
-
-  public BigDecimal getBidSize() {
-    return bidSize;
-  }
-
-  public BigDecimal getAskSize() {
-    return askSize;
-  }
-
-  public BigDecimal getDailyChange() {
-    return dailyChange;
-  }
-
-  public BigDecimal getDailyChangePerc() {
-    return dailyChangePerc;
-  }
-
-  public BigDecimal getLastPrice() {
-    return lastPrice;
-  }
-
-  public BigDecimal getVolume() {
-    return volume;
-  }
-
-  public BigDecimal getHigh() {
-    return high;
-  }
-
-  public BigDecimal getLow() {
-    return low;
-  }
-
-  public BigDecimal getAsk() {
-    return ask;
-  }
+  private Object placeHolder0;
+  private Object placeHolder1;
```

Comment:
do you have some example values this placeholder may have?
if so then please add then to your test

Reply:
these placeholders are reserved for the future and they are always null

Reply:
@walec51 these placeholders are reserved for the future and they are always null